frontend/packages/dev-console/src/components/add/DeveloperFeaturesGettingStartedCard.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { FlagIcon } from '@patternfly/react-icons';
+
+import { ALL_NAMESPACES_KEY, useActiveNamespace, useOpenshiftVersion } from '@console/shared/src';
+import {
+  GettingStartedLink,
+  GettingStartedCard,
+} from '@console/shared/src/components/getting-started';
+
+export const DeveloperFeaturesGettingStartedCard: React.FC = () => {
+  const { t } = useTranslation();
+  const [activeNamespace] = useActiveNamespace();
```

Comment:
i think `useActiveNamespace` is also used in  `SamplesGettingStartedCard` and both are being rendered in the `GettingStartedSection` component.
could we fetch the `activeNamespace` once in the `GettingStartedSection` and pass it on to `SamplesGettingStartedCard` and `DeveloperFeaturesGettingStartedCard.tsx` ?

Reply:
For now, I want to keep these cards as separate as possible. Maybe we load them with the extensions mechanism in the future as well.